Makefile.common

```diff
@@ -124,7 +124,7 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint yamllint unused build test
```

Comment:
Why we removed these two targets?

Reply:
in original exporter we don't have license headers, and I moved yaml as is.

Reply:
Did you check if this target was removed explicitly or if it was introduced later by upstream?
What do you mean by moving YAML as is? If we have something failing in our YAML then it's better to fix them.

Reply:
I moved configs from "old" pg exporter. It is a diff for upstream.